Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/TeleportPointer.cs

```diff
@@ -76,6 +77,26 @@ public class TeleportPointer : LinePointer, IMixedRealityTeleportPointer, IMixed
         /// </summary>
         public DistorterGravity GravityDistorter => gravityDistorter;
 
+        private float cachedInputThreshold = 0f;
+
+        private float inputThresholdSquared = 0f;
+
+        /// <summary>
+        /// The square of the InputThreshold value.
+        /// </summary>
+        private float InputThresholdSquared
+        {
+            get
+            {
+                if (Mathf.Approximately(cachedInputThreshold, inputThreshold))
```

Comment:
```suggestion
                if (!Mathf.Approximately(cachedInputThreshold, inputThreshold))
```

Reply:
i dont know how i failed to notice that.... THANKS!